.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+---
+name: "TER deployment"
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  set-matrix:
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - id: set-matrix
+        run: |
+          JSON=$(cat ./composer.json)
+          echo "::set-output name=${JSON//'%'/'%25'}"
```

Comment:
What does this job do? (I don't understand it yet …)

Reply:
This job get ext version from composer.json. In order to process json file we need to set it to matrix.